src/TimeCounter.cpp

```diff
@@ -3,21 +3,29 @@
 
 GJGameLevel* TimeCounter::currentLevel = nullptr;
 std::string TimeCounter::levelId;
+std::string TimeCounter::levelName;
 std::string TimeCounter::sessionId;
 
 void TimeCounter::setLevelId(GJGameLevel *level) {
     TimeCounter::levelId = std::to_string(level->m_levelID.value());
-
+    TimeCounter::levelName = std::string(level->m_levelName);
```

Comment:
Нельзя ли просто так сделать:
```suggestion
    TimeCounter::levelName = level->m_levelName;
```